services/galley/src/Galley/API/Teams.hs

```diff
@@ -182,8 +183,13 @@ updateTeamStatus tid (TeamStatusUpdate newStatus cur) = do
     Data.updateTeamStatus tid newStatus
   where
     journal Suspended _ = Journal.teamSuspend tid
-    journal Active c = Data.teamMembersUnsafeForLargeTeams tid >>= \mems ->
-      Journal.teamActivate tid mems c =<< Data.teamCreationTime tid
+    journal Active c = do
+      mems <- Data.teamMembersUnsafeForLargeTeams tid
+      teamCreationTime <- Data.teamCreationTime tid
+      -- TODO(Akshay): Figure out if this is required to be live data
+      -- a lot of tests assert this by using `createBindingTeam`
+      -- (TeamSize size) <- BrigTeam.getSize tid
```

Comment:
I'm not sure how to read this.  Line 191 has nothing to do with 190, no?  Should this be removed?

```suggestion
```

Reply:
I had something in my mind and forgot to act on it. Thankfully this confusing comment reminded me. I am not sure if member count is always 1 when a team is activated. I wanted to dig into that and it totally skipped my mind. I will look into this now.

Reply:
Better now?